gwtbootstrap3/src/main/java/org/gwtbootstrap3/client/ui/ListItem.java

```diff
@@ -65,6 +65,11 @@ public ListItem(final String text) {
      */
     @Override
     public void setText(final String text) {
+        if (this.text != null) {
```

Comment:
Would it be better/easier to just have the ```Text``` be created/added in the constructor then just set/get from there? Could you try that?